raiden/network/proxies/token_network.py

```diff
@@ -168,7 +168,7 @@ def new_netting_channel(
             'new_netting_channel called',
             peer1=pex(self.node_address),
             peer2=pex(partner),
-            channel_identifier=encode_hex(channel_identifier),
+            channel_identifier=channel_identifier,
```

Comment:
Do no do these changes here. It is happening in the PR for #2000

Reply:
This was only for testing, it's not related to this PR, so was meant to be dropped.

Reply:
Ah good thanks. I guess it's due to the dependency you have with raiden libs?

Reply:
Yes, exactly. As I need changes in raiden-libs (although, now they're on that PR which is pending review), so as it pulls raiden-libs, it breaks everything that isn't working on Raiden yet due to the contract changes.